averbis/core/_rest_client.py

```diff
@@ -139,6 +149,66 @@ def successful(self):
         return self.exception is None
 
 
+class ResourceContainer:
```

Comment:
I would love to see a __repr__ function on ResourceContainer like the following to make it easier to work with these objects.

```
 def __repr__(self):
        return f'{self.__class__.__name__}(name="{self.name}", scope="{self.scope}")'
```

`[<averbis.core._rest_client.ResourceContainer object at 0x7fa7ed9fac30>, <averbis.core._rest_client.ResourceContainer object at 0x7fa7ed9fa360>]` 

=> 

`[ResourceContainer(name="test-resource", scope="PIPELINE"), ResourceContainer(name="test-resource2", scope="PIPELINE")`